### DISCUSSION TITLE
Pinning a task to the top of a column

### DISCUSSION THREAD

**Discussion by erikshelley** [roovo/obsidian-card-board#168] [Show and tell]
I just discovered this plugin today and I love it! Hopefully we will get more control over task sorting in the future, but for now I discovered a neat little hack.

If you create a task with no due date and make it bold, it will show up at the top of its column since * comes before numbers and letters. In the attached image the pinned tasks are bold links. I use these pinned tasks as a sort of prompt to remind me of tasks I may need to add to each column.

![CardBoard](https://github.com/roovo/obsidian-card-board/assets/39465362/b244e83e-5c76-418f-b8c2-9a687a9b6610)

**Comment by roovo** [comment]
Hah - really interesting.  Love it when uninteneded features prove to be useful!

It would be good to add a supported way of pinning tasks tho :)
